### ISSUE TITLE
[5.0] Build fail in Windows 32bit.

### ISSUE THREAD

**Issue by NamsooCho** [Geal/nom#971]
```
error[E0277]: the trait bound `usize: util::cast::TryCast<i64>` is not satisfied
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\lexical-core-0.4.1\src\util\num.rs:34:14
   |
34 |         impl Number for $t {}
   |              ^^^^^^ the trait `util::cast::TryCast<i64>` is not implemented for `usize`
...
38 | number_impl! { u8 u16 u32 u64 usize i8 i16 i32 i64 isize f32 f64 }
   | ------------------------------------------------------------------ in this macro invocation
   |
   = help: the following implementations were found:
             <usize as util::cast::TryCast<f32>>
             <usize as util::cast::TryCast<f64>>
             <usize as util::cast::TryCast<i128>>
             <usize as util::cast::TryCast<i16>>
           and 6 others
error[E0277]: the trait bound `usize: util::cast::TryCast<usize>` is not satisfied
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\lexical-core-0.4.1\src\util\num.rs:34:14
   |
34 |         impl Number for $t {}
   |              ^^^^^^ the trait `util::cast::TryCast<usize>` is not implemented for `usize`
...
38 | number_impl! { u8 u16 u32 u64 usize i8 i16 i32 i64 isize f32 f64 }
   | ------------------------------------------------------------------ in this macro invocation
   |
   = help: the following implementations were found:
             <usize as util::cast::TryCast<f32>>
             <usize as util::cast::TryCast<f64>>
             <usize as util::cast::TryCast<i128>>
             <usize as util::cast::TryCast<i16>>
           and 6 others
error[E0277]: the trait bound `usize: util::cast::TryCast<u64>` is not satisfied
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\lexical-core-0.4.1\src\util\num.rs:34:14
   |
34 |         impl Number for $t {}
   |              ^^^^^^ the trait `util::cast::TryCast<u64>` is not implemented for `usize`
...
38 | number_impl! { u8 u16 u32 u64 usize i8 i16 i32 i64 isize f32 f64 }
   | ------------------------------------------------------------------ in this macro invocation
   |
   = help: the following implementations were found:
             <usize as util::cast::TryCast<f32>>
             <usize as util::cast::TryCast<f64>>
             <usize as util::cast::TryCast<i128>>
             <usize as util::cast::TryCast<i16>>
           and 6 others
error[E0277]: the trait bound `usize: util::cast::TryCast<u32>` is not satisfied
  --> C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\lexical-core-0.4.1\src\util\num.rs:34:14
   |
34 |         impl Number for $t {}
   |              ^^^^^^ the trait `util::cast::TryCast<u32>` is not implemented for `usize`
...
38 | number_impl! { u8 u16 u32 u64 usize i8 i16 i32 i64 isize f32 f64 }
   | ------------------------------------------------------------------ in this macro invocation
   |
   = help: the following implementations were found:
             <usize as util::cast::TryCast<f32>>
             <usize as util::cast::TryCast<f64>>
             <usize as util::cast::TryCast<i128>>
```

**Comment by kpp** [comment]
Appveyor build: https://ci.appveyor.com/project/tox-rs/tox/builds/25518335/job/67nafr85jd1u9aif

**Comment by Geal** [comment]
could you do a `cargo update` then try again? `lexical-core` 0.4.2 was released yesterday with a fix for 32 bit targets: https://github.com/Alexhuszagh/rust-lexical/blob/master/lexical-core/CHANGELOG#L16

**Comment by kpp** [comment]
It worked. Still the minimal version of lexical-core should be updated to 0.4.2

**Comment by Alexhuszagh** [comment]
@kpp Sorry for this, this should be patched in a more recent version of lexical, and the upstream issue is here:
https://github.com/Alexhuszagh/rust-lexical/issues/29

**Comment by Geal** [comment]
#1101 is merged, raising the minimal lexical-core to 0.6
